ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -38,7 +38,7 @@
   &__search-term-input {
     @include text_field;
     width: 80%;
-    min-height: calc($spacer__unit * 3);
+    min-height: calc($spacer__unit * 2);
```

Comment:
I have a question about this: in Chrome, I was finding that the $spacer__unit was not being interpolated correctly, so that all the instances of `calc()` containing $spacer__unit were failing to be calculated. I don't know loads about SASS so I had a google, and apparently it needs to be something like `calc(#{spacer__unit} * 2)` - is this correct in your experience?

Reply:
Thanks @lozette - I think this is the right syntax for `calc()` but when I saw your question I thought I might be out of date. I've just checked the Sass documentation and it does seem it's better not to avoid interpolation in calls to `calc()`. [This page](https://sass-lang.com/documentation/values/calculations) says:

> _You can also use interpolation in a calculation. However, if you do, nothing in the parentheses that surround that interpolation will be simplified or type-checked, so it’s easy to end up with extra verbose or even invalid CSS. Rather than writing `calc(10px + #{$var})`, just write `calc(10px + $var)`!_

In this case, the compiled CSS should have `min-height: 2rem` which seems to be what's generated by both `@debug` statements and the output. Is the field looking strange?

Reply:
I'm sorry my local is completely broken and I don't know how soon I can fix it, but IIRC Chrome isn't able to interpret anything where $spacer__unit is inside a calc(), and as a result things which are meant to have any spacing/padding using a calculated $spacer__unit is just not having any padding/spacing. There's an error in Chrome saying invalid declaration, I think.

Reply:
<img width="410" alt="Screenshot 2022-03-16 at 15 30 42" src="https://user-images.githubusercontent.com/1089521/158627469-ae34e12b-ec70-42f8-9c5c-7efb2b280bd5.png">

Finally got my local working again! Here's an example, if this makes sense.

Reply:
That's really strange. My Chrome console is fine (screenshot below). To be honest, I feel like it's fine - the `.scss` file is only available via a source map and isn't used by the browser to render. As long as the compiled CSS is fine and has the correct calculated properties it shouldn't pose a problem. I'll created a [Trello card](https://trello.com/c/hZX5RpzI) to take a closer look at this at a later date - we're using variables within `calc()` throughout the Sass. 

<img width="348" alt="Screenshot 2022-03-17 at 10 32 15" src="https://user-images.githubusercontent.com/16000203/158791408-b22df35d-d661-4baa-b06a-59fc6b470039.png">